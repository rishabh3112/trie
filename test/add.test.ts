import Store from "../src";

describe("add funtionality", () => {
  it("should add single data item into store", async () => {
    const store = new Store<string>("test");
    await store.add("rishabh", "https://rishabhchawla.co");

    const dataFromStore = await store.at("rishabh");
    expect(dataFromStore.data).toBe("https://rishabhchawla.co");
    expect(dataFromStore).toMatchSnapshot();
  });
});
