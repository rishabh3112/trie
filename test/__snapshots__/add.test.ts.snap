// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add funtionality should add single data item into store 1`] = `
Object {
  "data": "https://rishabhchawla.co",
  "id": "test#7",
  "key": "rishabh",
}
`;
